src/Microsoft.DotNet.Interactive.CSharpProject/ProjectFile.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Interactive.CSharpProject;
+
+public class ProjectFile
+{
+    public string Path { get; }
```

Comment:
Didn't we say we wanted to enforce this to be the `RelativeFilePath` type? An unvalidated `string` here will be a problem.